.travis.yml

```diff
@@ -28,6 +28,7 @@ install:
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coveralls
   - source activate test-environment
   - conda env update -n test-environment -f environment.yml
+  - conda install flake8
```

Comment:
shouldn't have to install flake8 -- can just install pre-commit and run `pre-commit run -a`.

Reply:
we could, my intention is to remove Travis as soon as the rest of it is working, so I went with the minimal change here.

Reply:
Ah yeah, fine.